lib/streamlit/config.py

```diff
@@ -375,6 +375,14 @@ def _global_development_mode() -> bool:
     type_=int,
 )
 
+_create_option(
+    "global.storeCachedForwardMessagesInMemory",
+    description="""If True, store cached ForwardMsgs in backend memory.""",
```

Comment:
nit: maybe add to this description that "this is an internal flag to validate a potential removal of the in-memory forward message cache". So that we don't forget later on what this flag is about.